bin/cli

```diff
@@ -54,7 +55,7 @@ drupal_install() {
   set -e
   time drush si "${OPTION}" \
     --sites-subdir=default \
-    --db-url=mysql://root:root@db:3306/"${OPTION}" \
+    --db-url=pgsql://root:root@"$DOCKER_NAME"_db:5432/"${OPTION}" \
```

Comment:
Can you set this back to db? It doesn't need to be the container name and will still work with db as is the name in the compose spec.